SPRINTLOG.md

```diff
@@ -458,3 +458,4 @@ _Nothing merged during this sprint_
 # 2024-11-18 – 2024-11-29
 
 - Logging: Add which user name reset password ([https://github.com/ScilifelabDataCentre/dds_web/pull/1574](https://github.com/ScilifelabDataCentre/dds_web/pull/1574))
+- Bugfix: Quick and dirty change to prevent `dds ls --tree` from failing systematically ([#1575](https://github.com/ScilifelabDataCentre/dds_web/pull/1575).
```

Comment:
Small thing but just because the final fixes and merge is done in this sprint.
```suggestion

# 2024-12-02 - 2024-12-13
- Bugfix: Quick and dirty change to prevent `dds ls --tree` from failing systematically ([#1575](https://github.com/ScilifelabDataCentre/dds_web/pull/1575).
```